tests/unit/customizations/s3/test_subcommands.py

```diff
@@ -132,6 +132,26 @@ def test_ls_command_with_no_args(self):
             's3', region_name=None, endpoint_url=None, verify=None,
             config=None))
 
+    def test_ls_command_with_non_existsent_url(self):
```

Comment:
Looks like there is a typo in the test name
```suggestion
    def test_ls_command_with_non_existent_url(self):
```